backend/src/main/java/com/example/demo/todo/TodoController.java

```diff
@@ -11,7 +11,8 @@
 
 @RestController
 @RequestMapping("/api/todos")
-@CrossOrigin
+//later delete it when it is in production
+@CrossOrigin(origins = {"http://localhost:3000"})
```

Comment:
When Spring Security is active, this annotation won't work anymore. You need to configure cors in the security config.